docs/guides/cluster-resources/configure-storage-and-rbac/examples/rbac.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: cluster-resource-reader
+  name: cluster-resource-reader-writer
   namespace: demo
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: cluster-resource-reader
+  name: cluster-resource-reader-writer
 rules:
 - apiGroups: ["*"]
   resources: ["*"]
-  verbs: ["get","list"]
+  verbs: ["*"]
```

Comment:
1) I am assuming `*` here is for GET, LIST, CREATE, UPDATE, PATCH. right?  Lets use those specific verbs in place of *.
2) We should explicitly say that this rbac has to be present in the cluster-2 beforehand, everytime we are creating the manifest restoreSession.